api/app/signals/settings/base.py

```diff
@@ -174,7 +174,8 @@
 
     AZURE_ACCOUNT_NAME = os.getenv('AZURE_STORAGE_ACCOUNT_NAME')
     AZURE_ACCOUNT_KEY = os.getenv('AZURE_STORAGE_ACCOUNT_KEY')
-    AZURE_URL_EXPIRATION_SECS = os.getenv('AZURE_STORAGE_URL_EXPIRATION_SECS', None)
+    AZURE_URL_EXPIRATION_SECS = int(os.getenv('AZURE_STORAGE_URL_EXPIRATION_SECS', 30*60))
+    AZURE_OVERWRITE_FILES = True # Make sure we do not get duplicate files in data warehouse
```

Comment:
This functionality is based on the name of a file, ```Overwrite an existing file when it has the same name as the file being uploaded. Otherwise, rename it. Default is False```. So if 2 or more reporters/users upload a file with the same name but different content the file will be overwritten. This is not the functionality we want. Please remove this from the PR.

Reply:
You are right, fixed this 👍